share/pot/stop.sh

```diff
@@ -48,40 +53,38 @@ _js_stop()
 			)
 		fi
 		_debug "Stop the pot $_pname"
-		touch "${POT_TMP:-/tmp}/pot_stopped_${_pname}"
 		jail -q -r "$_pname"
-		if [ -n "$_epair" ]; then
-			_debug "Remove ${_epair%b}[a|b] network interfaces"
-			sleep 1 # try to avoid a race condition in the epair driver,
-			        # potentially causing a kernel panic, which should
-			        # be fixed in FreeBSD 13.1:
-			        # https://cgit.freebsd.org/src/commit/?h=stable/13&id=f4aba8c9f0c
-			ifconfig "${_epair%b}"a destroy
-		else
-			if [ "$_network_type" = "alias" ]; then
-				_ip=$( _get_ip_var "$_pname" )
-				_debug "Remove $_ip aliases"
+	fi
+	# those are clean up operations for a pot already stopped
+	if [ -n "$_epair" ]; then
+		_debug "Remove ${_epair%b}[a|b] network interfaces"
+		sleep 1 # try to avoid a race condition in the epair driver,
+				# potentially causing a kernel panic, which should
+				# be fixed in FreeBSD 13.1:
+				# https://cgit.freebsd.org/src/commit/?h=stable/13&id=f4aba8c9f0c
+		ifconfig "${_epair%b}"a destroy
```

Comment:
`"${_epair%b}"a` will expand to `epair15aa` when called from start.

In general, I think passing the full interface name of the outer side in the -i parameter to this function makes sense and I would suggest to change this line to `ifconfig "${_epair}" destroy` and move epair interface name modifications up (see comment above).

Reply:
I'm changing the code to always consider the external epair (a) interface